samples/string_generator.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function stringGenerator (length, withSpaces) {
+	var chars = 'abcdefghijklmnopqrstuvwxyz';
```

Comment:
Fix this from 8-space indentation to 2-space, please.

Reply:
That's actually tabs :D But yeah - seems like we're not having a proper editorconfig in place for this repo. Changed.